docs/config.md

```diff
@@ -0,0 +1,71 @@
+# Integrate WSO2 API Manager 3.2.0 with Auth0
+
+In this guide, we explain how to integrate the WSO2 API Manager with an external Identity provider Auth0 as the OAuth Authorization Server 
+to manage the OAuth clients and tokens required by WSO2 API Manager. This is a client implementation that consumes APIs exposed by Auth0
+
+## Follow the instructions below to configure the third-party Key Manager
+
+### Step 1 : Prerequisites
+
+1.  Create an Auth0 account. Get the URL for the tenant. and then sign into the dashboard.
+
+   ![alt text](images/dashboard.png)
+
+
+2.  The you need to create an application to use the management API. Then need's to allow that application to use the management API.
+
+    ![alt text](images/management-api.png)
+
+    ![alt text](images/New%20Application.png)
+    
+    ![alt text](images/permision-to-use-app.png)
+   
+   Make sure you have granted all the permissions to Create, Manage Apps and Resource servers.    
+
+### Step 2: Configure WSO2 API Manager
+
+1.  Log into the admin portal of the API Manager. And add a new Key Manager.
+    
+    ![alt text](images/add-app-admin.png)
+    
+2.  Then select the Key manager type as Auth0 and provide the relevant fields accordingly.
+
+    ![alt text](images/km-tyoe.png)
+ 
+    **List of well know address could be found in advance section of the Auth0 Application settings**    
```

Comment:
Let's provide the sample values do that one can configure the connector in no time  :)

Reply:
This has changes done and docs updated